src/app/pages/home/containers/home-page.component.ts

```diff
@@ -22,6 +23,7 @@ import { HomePageAnimations } from './home-page.animation'
   animations: HomePageAnimations
 })
 export class HomePageComponent implements OnDestroy {
+  PROJECT_NAME = DefaultProjectName
```

Comment:
There is already support for Firebase remote config. Maybe we can get the project name from there if it exists otherwise fallback to the Default one.
I guess it will be parameterised based on `baseUrl`.